test/libsolidity/syntaxTests/immutable/conditionally_initialized.sol

```diff
@@ -1,9 +1,83 @@
 contract C {
-    uint immutable x;
+    uint immutable singleBranchWrite; // error
```

Comment:
The test case is comprehensive, but it would be nice to have one test case for each variable-test here, instead of one large one.
Easier to read, faster to see what fails, etc..

Reply:
Agree, done.